pepper-apis/dsm-core/src/main/resources/liquibase/singular/Pepper-195_geneticTestingUpdate.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="DDP-195_UPDATE_GENETIC_TESTING_ANSWER_FIELD" author="nick">
+        <sql>
+            UPDATE `field_settings` SET `actions` =
+            '[{\"type\":\"conditionalDisplay\",\"condition\":\"OTHER\",\"conditionalFieldSetting\":{\"columnName\":\"KARYOTYPE_ABNORMAL_OTHER_SPECIFY\",\"fieldType\":\"GROUP_GENETIC_TESTING\",\"columnDisplay\":\"Other Chromosomal anomaly (please specify)\",\"displayType\":\"TEXTAREA\",\"details\":{\"size\":100}}}]'
+            WHERE column_name='KARYOTYPE_ABNORMAL';
```

Comment:
I am wondering if update by `column_name` condition will work because we have a SAFE_UPDATE_MODE flag.

Reply:
I do not know what SAFE_UPDATE_MODE does but the migration did work on dev. Should I switch the `WHERE` to `field_settings_id`?

Reply:
If it worked then it's all good

Reply:
does it work?

Reply:
Yes, it worked for dev when I ran it. I have not tried it in any other environments.